sdl-core/src/main/scala/io/smartdatalake/statusinfo/StatusInfoListener.scala

```diff
@@ -0,0 +1,17 @@
+package io.smartdatalake.statusinfo
+
+import io.smartdatalake.app.StateListener
+import io.smartdatalake.config.SdlConfigObject.ActionId
+import io.smartdatalake.workflow.{ActionDAGRunState, ActionPipelineContext}
+
+class StatusInfoListener extends StateListener {
+
+  var stateVar: ActionDAGRunState = _
```

Comment:
I would recommend using Option[ActionDAGRunState = None. Otherwise it could result in NPEs if notifyState was not yet called...

Reply:
Done. Interestingly, the contents of the vars were null in the beginning, but it did not bother jetty. It caused the browser to say that the server did not respond and then display the result once data was available.
With the Options now, the server answers right away with just "null", which is replaced when data is available. I guess None gets serialized to null

Reply:
I prefer the behaviour with Option.